cb-talent-development/employer-style-base
# Remove @includes From flex Properties

**Purpose**
Removes `@includes` from `flex` properties

**Reasons:**
- Bourbon will deprecate these mixins: http://bourbon.io/docs/#flexbox
- [We only support IE11+](https://github.com/cbdr/employer#browser-testing), which supports standard properties, albeit with bugs: http://caniuse.com/#search=flexbox
- We have autoprefixer for our CSS output, which complains about the CSS output added by the mixins
![image](https://user-images.githubusercontent.com/2359538/30614310-99bb02fe-9d50-11e7-929c-20d1454428c8.png)
- It will also eliminate deprecation errors in Employer
![image](https://user-images.githubusercontent.com/2359538/30614338-b23aa050-9d50-11e7-8824-4ab553d1f884.png)
- Cleaner code

**JIRA**
n/a

**Changes**
* Improvements and fixes
 * 

* Changes to developer setup/environment
  * 

* Architectural changes
  * 

* Migrations or Steps to Take on Production
  * 
  
* Library changes
  * 

* Side effects
  * This likely breaks (or breaks *more*) layouts for IE10 and below

**Screenshots**
n/a

**Feature Server**
http://web.employer-6.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
n/a

* Steps to take
  * Check several random pages (**on IE11 and Safari in particular**) to make sure the lack of fallback properties doesn't break anything
  * There should be no differences from `develop`
  * Here's a list of one of each page type, with a few key/recent pages in bold

**http://web.employer-6.development.c66.me/**
**http://web.employer-6.development.c66.me/recruiting-solutions/post-your-job-right-now**
**http://web.employer-6.development.c66.me/recruiting-solutions/resume-database**
**http://web.employer-6.development.c66.me/recruiting-solutions/small-business-subscription-plans-and-pricing**
**http://web.employer-6.development.c66.me/company/why-careerbuilder**
**http://web.employer-6.development.c66.me/recruiting-solutions/applicant-tracking/dummy**

http://web.employer-6.development.c66.me/promotions
http://web.employer-6.development.c66.me/recruiting-solutions
http://web.employer-6.development.c66.me/dy-test
http://web.employer-6.development.c66.me/company/leadership
http://web.employer-6.development.c66.me/all-products-sitemap
http://web.employer-6.development.c66.me/employment-screening
http://web.employer-6.development.c66.me/portal/epwhg
http://web.employer-6.development.c66.me/recruiting-solutions/display-advertising
http://web.employer-6.development.c66.me/recruiting-solutions/talent-gather
http://web.employer-6.development.c66.me/recruiting-solutions/supply-and-demand
http://web.employer-6.development.c66.me/recruiting-solutions/post-job-ppc


* Responsive considerations
  * Check mobile devices to make sure the lack of fallback properties doesn't break anything

**Relevant PRs/Dependencies**
  * n/a

**Additional Information**
